File: EscritorioImagina/BIBLIOTECA_APP/Producto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BIBLIOTECA_APP
{
    public class Producto
    {
        
        //atributos
        public int id { get; set; }
        public String nombre { get; set; }
        public String descripcion { get; set; }
        public int precio { get; set; }
        public int stock { get; set; }
        public String foto { get; set; }

        public Producto()
        {

        }
    }
}

File: LibreriaImagina/src/java/Clases/Producto.java

package Clases;

import java.io.InputStream;


public class Producto {
    int id;
    String nombre;
    String descripcion;
    int precio;
    int stock;
    String foto;

    public Producto() {
    }

    public Producto(int id, String nombre, String descripcion, int precio, int stock, String foto) {
        this.id = id;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.precio = precio;
        this.stock = stock;
        this.foto = foto;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public int getPrecio() {
        return precio;
    }

    public void setPrecio(int precio) {
        this.precio = precio;
    }

    public int getStock() {
        return stock;
    }

    public void setStock(int stock) {
        this.stock = stock;
    }

    public String getFoto() {
        return foto;
    }

    public void setFoto(String foto) {
        this.foto = foto;
    }

}